types/coin.go

```diff
@@ -274,7 +274,8 @@ func (coins Coins) Validate() error {
 	}
 }
 
-func (coins Coins) isSorted() bool {
+// IsSorted returns true when coins are order ASC sorted with denoms.
+func (coins Coins) IsSorted() bool {
```

Comment:
this is api breaking we should add a changelog or we can add a new public fund function that does no validation to avoid breaking the api

Reply:
@tac0turtle Hi sir,  what does "api breaking" mean? `isSorted` function isn't public before, only used internally, I'm confused about "api breaking"

Reply:
true, sorry super jet lagged. still waking up